packages/react-hooks/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@aries-framework/react-hooks",
   "main": "build/index",
   "types": "build/index",
-  "version": "0.3.1",
+  "version": "0.3.2",
```

Comment:
This will be updated automatically by the CI.

It will actually be 0.4.0 as updating the  version of AFJ is a breaking change

Reply:
Reverted.